tools/shared.py

```diff
@@ -711,9 +715,7 @@ def set_logging():
 # Target choice. Must be synced with src/settings.js (TARGET_*)
 def get_llvm_target():
   if os.environ.get('EMCC_FAST_COMPILER') == '0':
-    if not os.environ.get('EMCC_LLVM_TARGET'):
-      os.environ['EMCC_LLVM_TARGET'] = 'le32-unknown-nacl'
-    return os.environ.get('EMCC_LLVM_TARGET')
+    return 'unavailable-non-fastcomp'
```

Comment:
Because it means we crash as soon as we import shared (this function is run on import). Which would be fine, except for `--help` and `--version`.